addons/web/static/src/js/views/view_dialogs.js

```diff
@@ -129,7 +134,7 @@ var FormViewDialog = ViewDialog.extend({
                     }
                 });
 
-                if (multi_select) {
+                if (multi_select && this.activeActions.create) {
```

Comment:
You have been spreading this "activeActions" stuff everywhere, but I'm not sure I understand why do you need it ?

Reply:
It's just a better check, to make sure that buttons are displayed only when the related actions are permitted / allowed.

Reply:
Ok but what cases do not work if we remove it ? If this solves is a bug in standard then it's worth its own commit.

Reply:
Nope it's not solving any bug. We can remove these changes.